### DISCUSSION TITLE
Build glib-2.73.3 universal library(arm64 and x86_64) on macOS

### DISCUSSION THREAD

**Discussion by diuming** [Homebrew/discussions#3615] [Everyday usage]
OS: macOS Monterey 12.5.1 with Apple M1 chip

How do i use meson to build glib universal library(arm64 and x86_64)?
Is it possible to build glib fat library?

I try to build for arm64 is ok, but for x86_64 always failure.

Could you give me a advice? thanks.

```
meson x86_64-build  --prefix=/some_place/x86_64/glib-2.73.3 --cross-file /tmp/x86_64-darwin.txt --default-library=static --buildtype=release

/tmp/x86_64-darwin.txt
-------------------------
[host_machine]
system = 'darwin'
cpu_family = 'x86_64'
cpu = 'x86_64'
endian = 'little'

[binaries]
c = 'clang'
cpp = 'clang++'
objc = 'clang'
ld = 'ld'
strip = 'strip'

```

**Comment by carlocab** [comment]
This seems like a question better directed at the Meson project. Please check their documentation.
